plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformUtils.kt

```diff
@@ -196,7 +196,7 @@ suspend fun JobId.pollTransformationStatusAndPlan(
                     CodetransformTelemetry.jobStatusChanged(
                         codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                         codeTransformJobId = this.id,
-                        codeTransformStatus = newStatus.toString()
+                        codeTransformStatus = "$state -> $newStatus", // old status -> new status
```

Comment:
Useful so that we can track the old state and new state that a job transitioned to.

Reply:
Blocker: I was expecting a new field. This is not backwards compatible with the current `codeTransformStatus` emission. With this change we would also have to parse this field to get the status. It will be more difficult to visually graph this.

Reply:
Also this change would need to be reflected in VSCode to have telemetry parity.

Reply:
Makes sense. Will add a new field and yes this is planned for VS Code too.

Reply:
This will be done in a separate PR